### PR TITLE
Replacing variables with environment variables

### PR DESCRIPTION
**Pull request by IvanGodinez21**
I would like to use .env file, so I edited the code and I think it could be used by other users, and maybe they may like it:
This allows to use the program in hosting platforms (Like Heroku).

Now requires the following modules:
-dotenv
-env-smart

This is just a suggestion, and this might require some changes to work perfectly in hosting platforms

### CODE REVIEW

File: src/spotify.service.ts [hunk 1]
```diff
@@ -1,19 +1,21 @@
 import SpotifyWebApi from 'spotify-web-api-node';
 import { waitForCode } from './auth-server';
-import config from './config.json';
+import 'dotenv/config'
+import env from 'env-smart';
 import SpotifyAuth from './spotify-auth';
 import fs from 'fs';
 import { response } from 'express';
+env.load();
```
Comment (MarcDonald) [inline src/spotify.service.ts:9]: Would it be possible to have all the necessary config declared in the imports like this
`const { SPOTIFY_CLIENT_ID, SPOTIFY_CLIENT_SECRET, etc } = process.env`
Personally I think it makes it look a bit nicer just so there isn't a million `process.env.XYZ` spread throughout the code

**Review by MarcDonald — CHANGES_REQUESTED** [review]
Hey, this is really cool thanks for doing it! It's way better than my `config.json` approach.

Just left a comment on one or two things that I noticed while reading through the code, if these could be addressed I'll go ahead and approve and merge it 🙂

**Comment by IvanGodinez21** [comment]
Done, I fixed everything based on your comments @MarcDonald
_I think 😅_

**Review by MarcDonald — APPROVED** [review]
Looks good

**Comment by IvanGodinez21** [comment]
_Sorry I can't merge, only those with write access to this repository can merge pull requests._

Also, I have found that some platforms start the deployment running the "build" script, if you wish to run it without problems, you should change the script section of **package.json** like this:

"scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "build": "npm run tsc && node build/index.js",
    "tsc": "tsc",
    "start": "npm run tsc && node build/index.js",
    "start:dev": "nodemon --watch 'src/**/*.ts' --exec 'ts-node' src/index.ts",
    "lint": "prettier -c 'src/**/*{.js,.ts}' && prettier -c '*.md'",
    "lint:fix": "prettier --write 'src/**/*{.js,.ts}' && prettier --write '*.md'"
  },

**Comment by MarcDonald** [comment]
My bad, I'll merge this now and make that change the next time I do a commit. Thanks! 😄

**Pull request merged**